### DISCUSSION TITLE
Inscrire le nom et le prénom d'un client issus d'une BD dans un texte <p>

### DISCUSSION THREAD

**Discussion by GigMatt** [MTESSDev/FRW#100] [Annonces]
Allo, 
Nous voudrions pousser l'information que nous avons dans un champ. Voici notre idée:
![image](https://github.com/MTESSDev/FRW/assets/126168415/77bc5640-e622-4e1a-a59a-344fca01767e)

Pour se faire, nous avons du placer un code caché dans le computed
![image](https://github.com/MTESSDev/FRW/assets/126168415/682f5ed5-1127-452f-9a99-1f97a29615e5)

Est-ce une bonne pratique?

**Comment by vinnyboii** [comment]
Bonjour,

Si je comprends bien, votre besoin est de pré remplir certains champs lors de la création du formulaire afin que le citoyen n'ait pas à saisir de l'information déja connue par votre système?

Il s'agit de la notion de préremplissage que nous expliquons ici: https://github.com/MTESSDev/FRW/blob/main/Documentation/pre-remplissage.md

Reviens moi si tu as des questions.

**Reply by vinnyboii** [reply]
@j0ber time to shine

**Reply by GigMatt** [reply]
@vinnyboii , nous testons avec le code javascript dans "computed" et te revenons, merci!

**Comment by j0ber** [comment]
Bonjour,

Nul besoin d'injecter du script dans votre cas... 

Nous avons ajouté un exemple d'injection de valeur d'un champ du modèle dans un texte avec les explications nécessaires.

Voir l'exemple 4 disponible ici : https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=sectionStyle

Il pourrait être utile d'injecter du script dans cette situation, seulement s'il y avait un traitement à faire avec la valeur du modèle pour l'afficher.

**Reply by GigMatt** [reply]
Juste parfait    {{val('j0ber')}}      Merci!
